index.ts

```diff
@@ -119,24 +162,36 @@ export async function exportEventsToBigQuery(events: PluginEvent[], { global }:
                 elements = $elements
             }
 
-            return {
-                uuid,
-                event: eventName,
-                properties: JSON.stringify(ingestedProperties || {}),
-                elements: JSON.stringify(elements || {}),
-                set: JSON.stringify($set || {}),
-                set_once: JSON.stringify($set_once || {}),
-                distinct_id,
-                team_id,
-                ip,
-                site_url,
-                timestamp: timestamp ? global.bigQueryClient.timestamp(timestamp) : null,
+            const object: {json: Record<string, any>, insertId?: string} = {
```

Comment:
Can we simplify this object if not using insertId?

Reply:
I could remove the `insertId` from the type, but other than that, nope, can't do, because the sdk by default inserts a random `insertId` unless I'm using the raw version, which requires the object to look like this^